packages/fftw3/fftw3.0.7.1/opam

```diff
@@ -1,10 +1,10 @@
-opam-version: "1"
-name: "fftw3"
-version: "0.7.1"
+opam-version: "1.2"
 authors: ["Christophe Troestler <Christophe.Troestler@umons.ac.be>"]
 maintainer: "Christophe.Troestler@umons.ac.be"
 license: "LGPL-2.1 with OCaml linking exception"
 homepage: "https://github.com/Chris00/fftw-ocaml"
+dev-repo: "https://github.com/Chris00/fftw-ocaml.git"
+bug-reports: "https://github.com/Chris00/fftw-ocaml/issues"
 build: [
   ["./configure" "--prefix" prefix "--sbindir=%{lib}%/fftw3/sbin" "--libexecdir=%{lib}%/fftw3/libexec" "--sysconfdir=%{lib}%/fftw3/etc" "--sharedstatedir=%{lib}%/fftw3/com" "--localstatedir=%{lib}%/fftw3/var" "--libdir=%{lib}%/fftw3/lib" "--includedir=%{lib}%/fftw3/include" "--datarootdir=%{lib}%/fftw3/share"]
   [make]
```

Comment:
On Mon, 6 Jul 2015 05:26:57 -0700, David Sheets wrote:
> 
> [make "install"] could go into the install section but I've merged this anyway.

Right.  Will be for the next version.